### ISSUE TITLE
(Accessibility) The close button does not have a text alternative, and should be a button

### ISSUE THREAD

**Issue by bryclee** [paypal/paypal-checkout#370]
The following were raised as accessibility issues:
1. Close button does not have a text alternative
    - Fix would be to add `aria-label` with appropriate description (eg. "Close") to the element
2. Close button should be an HTML button or have `role=button`
    - This is so that it can present the appropriate role to the user through screen readers
    - In addition, buttons should be interactable with the `space` and `enter` buttons

**Comment by bluepnume** [comment]
Thanks! Will take a look

**Comment by bluepnume** [comment]
Fix is in.
